Cask

```diff
@@ -5,4 +5,6 @@
 
 (development
  (depends-on "noflet")
- (depends-on "helm"))
+ (depends-on "ag")
+ (depends-on "helm")
+ (depends-on "helm-ag"))
```

Comment:
`helm-ag` is a package that doesn't having anything to do with `ag` or `projectile`. This is what I was pointing out.